build.sh

```diff
@@ -39,7 +39,8 @@ docker image build ./$IMAGE_DIR --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME --tag
 
 rm -f /home/jenkins/.docker/config.json
 
-if [ $IMAGE_DIR = "terra-jupyter-gatk" ]; then
+if [[ $IMAGE_DIR = "terra-jupyter-gatk" || \
```

Comment:
I don't think we need to push gatk images anymore to dockerhub?

Reply:
Done